aten/src/ATen/native/quantized/cpu/fused_obs_fake_quant.cpp

```diff
@@ -220,6 +220,9 @@ at::Tensor fused_moving_avg_obs_fake_quant(
     const int64_t ch_axis,
     bool per_row_fake_quant,
     bool symmetric_quant) {
+  if (self.numel() == 0) {
+    return self.clone();
```

Comment:
why not return `self` directly?

Reply:
In PyTorch only composite ops (ops that we don’t worry about for autograd/dispatch) are allowed to return view sometimes and other times copies. If we return x directly it may also cause unexpected behavior in the backward pass because now autograd doesn’t know that this is actually a view of the same storage and might mess up the gradients.

Since this op is not a composite op, we return the clone to avoid any unexpected behavior.